byron/ledger/executable-spec/test/Ledger/Delegation/Examples.hs

```diff
@@ -55,17 +54,17 @@ deleg =
     ]
 
   , testGroup "Scheduling"
-    [ testCase "Example 0" $ checkTrace @SDELEG (DSEnv [gk 0, gk 1, gk 2] (e 8) (s 2) (bc 5)) $
+    [ testCase "Example 0" $ checkTrace @SDELEG (DSEnv [gk 0, gk 1, gk 2] (e 8) (s 2)) $
 
       pure (DSState [] [])
 
-    .- dc (gk 0) (k 10) (e 8) .-> DSState [(s 12, (gk 0, k 10))]
+    .- dc (gk 0) (k 10) (e 8) .-> DSState [(s 4322, (gk 0, k 10))]
```

Comment:
What's the motivation behind this change? 4322 seems like a very strange number to hard-code

Reply:
It's not any different than 12. I could use someone's help here as I don't fully understand the test.

Reply:
The tests are just testing that given the initial states, and the signals, we get the corresponding final states. These are just random numbers, but I think @nc6 's point was that if we start with some arbitrary random numbers, there shouldn't be a reason for changing them to some other numbers. 

Also if you have change a value in the signal you'd have to update the other values that come after that signal.

Reply:
I'm not sure I'm following. What needs to be done with this test (if anything)?

Reply:
Nothing. Just revert back 4332 to 12, or change the ensuing states accordingly.